react/features/filmstrip/actions.native.js

```diff
@@ -60,3 +61,20 @@ export function setTileViewDimensions({ thumbnailSize }: Object) {
         }
     };
 }
+
+/**
+ * Sets the list of the reordered remote participants based on which the visible participants in the filmstrip will be
+ * determined.
+ *
+ * @param {Array<string>} participants - The list of the remote participant endpoint IDs.
+ * @returns {{
+        type: SET_REMOTE_PARTICIPANTS,
+        participants: Array<string>
+    }}
+ */
+export function setRemoteParticipants(participants: Array<string>) {
```

Comment:
nit: alpha sort, should be inserted before `setTileViewDimensions` function.

Reply:
Also, don't we want to move `setRemoteParticipants` to a common file actions.any.js. ?

Reply:
will move it to a common file! Thanks for the idea!